sys-auth/libfprint/libfprint-1.90.1.ebuild

```diff
@@ -10,23 +10,24 @@ HOMEPAGE="https://cgit.freedesktop.org/libfprint/libfprint/ https://github.com/f
 SRC_URI="https://gitlab.freedesktop.org/libfprint/libfprint/-/archive/v${PV}/${PN}-v${PV}.tar.gz -> ${P}.tar.gz"
 
 LICENSE="LGPL-2.1"
-SLOT="0"
+SLOT="2"
```

Comment:
requires a revbump

Reply:
...and a blocker on the previous version since the new slot's going to collide with it. Or maybe a slotmove but I'm not sure about partial moves.

Reply:
Okay, all slot changes got a revbump and blockers are added.